server/server.js

```diff
@@ -4,6 +4,7 @@ const cors = require('cors')
 const server = express()
 const { v4: uuidv4 } = require('uuid');
 let map = new Map(); //creating map for uuid as key and token as value
+var mainUser; //creating user
```

Comment:
Please use `let` or `const` instead of `var`. This is an old type of variable declarations.

Reply:
done